### PR TITLE
Update photo-sphere-viewer.js to v4

### PR DESCRIPTION
**Pull request by R0Wi**
Closing #1

### CODE REVIEW

File: tests/Testdata/pos1.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+	"fullWidth": 13934,
```
Comment (bahnwaerter) [inline tests/Testdata/pos1.json:2]: Please indent the lines evenly.

```suggestion
  "fullWidth": 13934,
```

File: tests/Testdata/pos10.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+	"fullWidth": 6032,
```
Comment (bahnwaerter) [inline tests/Testdata/pos10.json:2]: Please indent the lines evenly.

```suggestion
  "fullWidth": 6032,
```

File: tests/Testdata/pos2.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+	"fullWidth": null,
```
Comment (bahnwaerter) [inline tests/Testdata/pos2.json:2]: Please indent the lines evenly.

```suggestion
  "fullWidth": null,
```

File: tests/Testdata/pos3.json [hunk 4]
```diff
@@ -0,0 +1,11 @@
+{
+	"fullWidth": 21243,
```
Comment (bahnwaerter) [inline tests/Testdata/pos3.json:2]: Please indent the lines evenly.

```suggestion
  "fullWidth": 21243,
```

**Review by bahnwaerter — CHANGES_REQUESTED** [review]
Everything looks fine to me except some minor indentation errors. Please fix the incorrect indentations. Note that I only reviewed the code and haven't done any manual testing yet.

File: tests/Testdata/pos3.json (hunk 4 above)
Comment (R0Wi) [reply tests/Testdata/pos3.json]: A bit unfair since VSCode shows them all aligned :smile: Reformatted ...

**Comment by R0Wi** [comment]
@bahnwaerter i did a rebase on the NC24 compatible version and forced pushed including the requested changes. I think we have to manually backport this to NC23 stable in a separate PR (see https://github.com/nextcloud/files_photospheres/pull/89)

**Pull request merged**